### PR TITLE
Adding mujoco recipe

### PR DESCRIPTION
**Pull request by lmilechin**
Adding "How To" recipe for installing and using Mujoco on both Engaging and SuperCloud

### CODE REVIEW

**Comment by christophernhill** [comment]
@lmilechin we could skip Satori for this if you want? Not sure if there is a Power9 variant?

**Comment by christophernhill** [comment]
@lmilechin what do you think about a little preamble of form


MuJoCo is a free and open source physics engine that aims to facilitate research and development in robotics, biomechanics, graphics and animation, and other areas where fast and accurate simulation is needed.

You can learn about mujoco here https://mujoco.org

**Comment by christophernhill** [comment]
@lmilechin this looks awesome. 
I have a thought though! 

Should we use virtualenv for both environments?  I was talking with NSE folks about this 

https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/

yesterday, which suggests this sort of recipe as a "standard" approach (as of today!) 
 
```
   python3 -m pip install --user --upgrade pip
   python3 -m pip install --user virtualenv
   python3 -m venv env
   source env/bin/activate
   python3 -m pip install requests
  .....
```

What do you think? 

Presumably "standard" approach will continue to evolve - but  doing pip as `python3 -m pip ` could be a good thing? It should mean pip and the python version are in sync?

**Comment by christophernhill** [comment]
@lmilechin should we try adding this

https://github.com/zhaoterryy/mkdocs-git-revision-date-plugin

I can test it if you think it might be useful? 
I think it would automate adding a last committed/updated date slug on a page, if t works.

File: docs/recipes/mujoco.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+# Installing and Using MuJoCo
```
Comment (christophernhill) [inline docs/recipes/mujoco.md:1]: @lmilechin should we add this at the top? 
---
tags:
 - Engaging
 - SuperCloud
 - Howto Recipes
 - MuJoCo
---

**Comment by lmilechin** [comment]
> @lmilechin should we try adding this
> 
> https://github.com/zhaoterryy/mkdocs-git-revision-date-plugin
> 
> I can test it if you think it might be useful? I think it would automate adding a last committed/updated date slug on a page, if t works.

Yes, that looks useful!

**Review by christophernhill — APPROVED** [review]
@lmilechin looks good -

**Comment by lmilechin** [comment]
I've updated the Engaging instructions. It still would not build on Rocky, it looks like we are missing the X11 header file. But switching to an anaconda module on the usual login nodes worked well, I didn't need the additional dependency so I removed that part.

**Pull request merged**